src/components/InstancesList/InstancesList.js

```diff
@@ -967,6 +968,7 @@ class InstancesList extends React.Component {
       if (isBrowseOption) {
         parentMutator.browseModeRecords.reset();
         this.setState({ isSingleResult: false });
+        goTo(path);
```

Comment:
Will this also close the record when switching from one browse option to another browse option?

Reply:
I was worrying  about that too. But the list items don't open any record, they redirect user to non browse options. Correct me If I am wrong